### ISSUE TITLE
CurseForge/Modrinth release?

### ISSUE THREAD

**Issue by ghost** [Asek3/sodium-1.12#50]
### Request Description

Currently, the only way to download Vintagium is via GitHub actions, which isn't exactly user-friendly. It'd be much easier if the mod was available on Modrinth or CurseForge.

**Comment by duckopia** [comment]
No, you can't. You need to wait until the Modder decide to do it. If you publish this file to CurseForge/Modrinth with no Modder's permission. Your content maybe takedown by DMCA.

**Comment by ghost** [comment]
I know that. I never said I wanted to put the mod on CF or MR myself, I only wanted to know if it's even planned.

**Comment by duckopia** [comment]
I don't know when, but if this mod finished, he will release it to CurseForge/Modrinth.
> I know that. I never said I wanted to put the mod on CF or MR myself, I only wanted to know if it's even planned.

**Comment by duckopia** [comment]
If you also a minecraft modder, you can help him. This mod will release faster with the help of the community.

**Comment by Dvow** [comment]
> ### Request Description
> Currently, the only way to download Vintagium is via GitHub actions, which isn't exactly user-friendly. It'd be much easier if the mod was available on Modrinth or CurseForge.

is anyone able to help me getting this mod to work?

**Comment by duckopia** [comment]
> > ### Request Description
> > Currently, the only way to download Vintagium is via GitHub actions, which isn't exactly user-friendly. It'd be much easier if the mod was available on Modrinth or CurseForge.
> 
> is anyone able to help me getting this mod to work?

https://github.com/Asek3/sodium-1.12/actions/runs/11309063708
You can download it from the Github Actions

**Comment by Dvow** [comment]
> > > ### Request Description
> > > Currently, the only way to download Vintagium is via GitHub actions, which isn't exactly user-friendly. It'd be much easier if the mod was available on Modrinth or CurseForge.
> > 
> > 
> > is anyone able to help me getting this mod to work?
> 
> https://github.com/Asek3/sodium-1.12/actions/runs/11309063708 You can download it from the Github Actions

is there any dependancies to run it?

**Comment by duckopia** [comment]
> > > > ### Request Description
> > > > Currently, the only way to download Vintagium is via GitHub actions, which isn't exactly user-friendly. It'd be much easier if the mod was available on Modrinth or CurseForge.
> > > 
> > > 
> > > is anyone able to help me getting this mod to work?
> > 
> > 
> > https://github.com/Asek3/sodium-1.12/actions/runs/11309063708 You can download it from the Github Actions
> 
> is there any dependancies to run it?

You also need to install MixinBooter:
CurseForge: https://www.curseforge.com/minecraft/mc-mods/mixin-booter 
or
Modrinth: https://modrinth.com/mod/mixinbooter

If you don't install it, Minecraft Forge 1.12.2 will not work.
I don't know why, but it's the way that I usually use to make this mod working.
There is also a list of Vintagium-Compatibility-List: https://github.com/daizu-007/Vintagium-Compatibility-List.
It maybe helpful for you.

**Comment by Levviata** [comment]
[Cleanroom's Discord](https://discord.gg/f2K4aSpG4F) server has a compiled jar from commit 29dc4a8d527c8a22dfca4c5ebce82303df2a0615 [here](https://discord.com/channels/926486493562814515/1185366081133617293/1259432738843398265) (or check '#information' yourself Lol), use that (or don't, know you have it, posting for posterity)

**Comment by ghost** [comment]
Closing since forks are available on CF/MR  and Celeritas is now a thing as well.
